src/wallet3/db_schema.cpp

```diff
@@ -318,7 +319,19 @@ namespace wallet
 
     while (st->executeStep())
     {
-      outs.emplace_back(db::get<int64_t, int64_t, int64_t, int64_t, int64_t, int64_t, int64_t>(st));
+      auto& out = outs.emplace_back();
```

Comment:
`Ouput` needs a constructor that takes a tuple, then you can do:

`outs.emplace_back(db.get<int64_t, int64_t, int64_t, int64_t, int64_t, std::string,
           std::string, std::string, int64_t, int64_t>(st));`

and implement the rest in the constructor of `Outputs`